### PR TITLE
Updated C# SDK; use the correct xapi tag for vTPM support. 

### PR DESCRIPTION
**Pull request by kc284**

### CODE REVIEW

**Review by danilo-delbusso — APPROVED** [review]

**Comment by kc284** [comment]
I forgot to say this PR is easier to review with whitespace off.

File: XenModel/XenAPI/VTPM.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ namespace XenAPI
 {
     /// <summary>
     /// A virtual TPM device
-    /// First published in XenServer 4.0.
+    /// First published in .
```
Comment (CitrixChris) [inline XenModel/XenAPI/VTPM.cs:45]: this comment doesn't contain anything useful

Comment (CitrixChris) [reply XenModel/XenAPI/VTPM.cs:45]: I'm assuming its meant to be 22.26.0-next

File: XenModel/XenAPI/VTPM.cs [hunk 2]
```diff
@@ -361,7 +361,7 @@ public static List<XenRef<VTPM>> get_all(Session session)
 
         /// <summary>
         /// Get all the VTPM Records at once, in a single XML RPC call
-        /// First published in XenServer 4.0.
+        /// First published in .
```
Comment (CitrixChris) [inline XenModel/XenAPI/VTPM.cs:364]: first published when?

**Review by CitrixChris — CHANGES_REQUESTED** [review]

**Review by CitrixChris — APPROVED** [review]

**Pull request merged**